### ISSUE TITLE
Build failed when running scripts

### ISSUE THREAD

**Issue by xmkevin** [BoltsFramework/Bolts-ObjC#295]
/Users/admin/Downloads/Bolts-ObjC-1.8.4/Bolts/iOS/BFAppLinkReturnToRefererView.m:33:1: error:
      cannot synthesize weak property in file using manual reference counting
@implementation BFAppLinkReturnToRefererView {
^
In file included from /Users/admin/Downloads/Bolts-ObjC-1.8.4/Bolts/iOS/BFAppLinkReturnToRefererView.m:11:
/Users/admin/Downloads/Bolts-ObjC-1.8.4/Bolts/iOS/BFAppLinkReturnToRefererView.h:56:70: note:
      property declared here
@property (nonatomic, weak) id<BFAppLinkReturnToRefererViewDelegate> delegate;
                                                                     ^
1 error generated.

The following build commands failed:
	CompileC build/Bolts.build/Debug-iphonesimulator/Bolts-iOS.build/Objects-normal/i386/BFAppLinkReturnToRefererView.o Bolts/iOS/BFAppLinkReturnToRefererView.m normal i386 objective-c com.apple.compilers.llvm.clang.1_0.compiler
(1 failure)

**Comment by opoukha** [comment]
Same thing for 1.8.3

**Comment by nlutsenko** [comment]
You do need to run `git submodule update --init --recursive` after you cloned the repository.
We use submodules for a shared set of configurations of Xcode config files, which also includes the configuration on whether to enable ARC (which is the error that you see here).

TL;DR Run this to unbreak:
```bash
git clone https://github.com/BoltsFramework/Bolts-ObjC.git
cd Bolts-ObjC
git submodule update --init --recursive
```
